### PR TITLE
Update README documentation

### PR DESCRIPTION
**Pull request by lucasmelogithub**
- Add "Modules" language

### CODE REVIEW

**Review by JoshHilliker — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -20,24 +20,21 @@
 6. Ready to be Integrated into Intel® Cloud Optimization Modules for HashiCorp Terraform using Cloud Cloud-Init integration. See [GCP Module example](https://github.com/intel/terraform-intel-gcp-vm/tree/main/examples/gcp-linux-with-aikit)).
 7. Can also be used directly on their own(see usage on each recipe).
 
-**Currently out-of-scope for OCRs:**
+**Currently out-of-scope:**
 
-1. Physical on-prem hardware configuration.
-2. Defining tuning parameters/knobs and values that are required to optimize a workload.
-3. Benchmarking.
+1. Physical on-prem hardware configuration
```
Comment (wsfowler) [inline README.md:25]: We have examples of this in the repo.

File: README.md [hunk 2]
```diff
@@ -190,4 +187,4 @@ We are glad to review and accept contributions.
 
 Today, the scope is limited to enabling/optimizing VMs on public cloud providers.
```
Comment (wsfowler) [inline README.md:188]: Can we modify this a bit, as I think we'll want to use this repo to provide some help with Gaudi configurations.

File: README.md [hunk 3]
```diff
@@ -190,4 +187,4 @@ We are glad to review and accept contributions.
...
 
-The OCRs are currently not managing on-prem physical hardware configurations.
+The Intel Cloud Optimized Modules for Ansible are currently not managing on-prem physical hardware configurations.
```
Comment (wsfowler) [inline README.md:190]: See other comments.

**Pull request merged**